crates/rust-analyzer/Cargo.toml

```diff
@@ -38,7 +38,7 @@ tracing = "0.1"
 tracing-subscriber = { version = "0.2", default-features = false, features = ["env-filter", "registry"] }
 tracing-tree = { version = "0.1.4" }
 always-assert = "0.1"
-
+num-format = "0.4.0"
```

Comment:
We don't use helper crates:

https://github.com/matklad/rust-analyzer/blob/e4b6ca47e458261251e390953f71f0efafdd2a82/docs/dev/style.md#L53-L61

If you need some special formating infra, implement it in this repo. See, for example, the Bytes type:

https://github.com/matklad/rust-analyzer/blob/e4b6ca47e458261251e390953f71f0efafdd2a82/crates/profile/src/memory_usage.rs#L90-L115

Reply:
Oh sorry, my bad. Let me fix.